scripts/get

```diff
@@ -111,8 +111,10 @@ downloadFile() {
   HELM_DIST="helm-$TAG-$OS-$ARCH.tar.gz"
   DOWNLOAD_URL="https://kubernetes-helm.storage.googleapis.com/$HELM_DIST"
   CHECKSUM_URL="$DOWNLOAD_URL.sha256"
-  HELM_TMP_FILE="/tmp/$HELM_DIST"
-  HELM_SUM_FILE="/tmp/$HELM_DIST.sha256"
+  HELM_TMP_ROOT="/tmp/$(whoami)"
```

Comment:
I think it's a little concerning deleting an entire directory in /tmp. There could be other processes that use this directory as well because the name is not unique enough.

If we're going to be cleaning up the directory then it's probably better if we actually used `mktemp -d` instead of `/tmp/$(whoami)` to create a temporary directory. That way we can write to it, create it and eventually destroy it without being concerned about destroying files that other processes use.

Reply:
TIL [`mktemp`](https://www.gnu.org/software/autogen/mktemp.html#TOP) 👍